execute.php

```diff
@@ -50,6 +50,92 @@
   $hostname = trim($_POST['routers']);
   $parameter = trim($_POST['parameter']);
 
+  
+  /// Rate limit
+  $limit = 3; //limit command
```

Comment:
These variables should probably be configurable by users. So they should be configuration options.